pkg/config/env_var_test.go

```diff
@@ -192,29 +199,68 @@ func TestRemoveEnvVarsFromList(t *testing.T) {
 			},
 		},
 		{
+			name: "Case 4: Check failing with foo=bar (invalid variable.. should error out)",
+			envVarList: EnvVarList{
+				EnvVar{
+					Name:  "foo",
+					Value: "bar",
+				},
+				EnvVar{
+					Name:  "foo",
+					Value: "bar",
+				},
+			},
+			expected: EnvVarList{},
+			keys:     []string{"foo=bar", "hi=hello"},
+			wantErr:  true,
+		},
+		{
+			name: "Case 5: Check failing when passing in multiple vals but one is valid",
 			envVarList: EnvVarList{
 				EnvVar{
 					Name:  "foo",
 					Value: "bar",
 				},
+				EnvVar{
+					Name:  "hi",
+					Value: "hello",
+				},
+			},
+			expected: EnvVarList{},
+			keys:     []string{"foo=bar", "hi"},
```

Comment:
Shouldn't this pass because although the env var `foo=bar` doesn't exist, `hi` does exist.

Reply:
No, because we fail at foo=bar immediately before returning the whole list. We shouldn't return a pass when one of the parameters is incorrect.